app/elm/Extra/GraphQL.elm

```diff
@@ -1,18 +1,31 @@
-module Extra.GraphQL exposing (oneOf)
+module Extra.GraphQL exposing (postOperationSimple)
 
-import Graphql.SelectionSet
+import GraphQL.Errors
+import GraphQL.Operation
+import GraphQL.Response
+import Http
 
 
-oneOf : List (Graphql.SelectionSet.SelectionSet (Maybe a) scope) -> Graphql.SelectionSet.SelectionSet a scope
-oneOf selectors =
-    selectors
-        |> Graphql.SelectionSet.list
-        |> Graphql.SelectionSet.mapOrFail
-            (getFirstNonNull
-                >> Result.fromMaybe "Expecting one of the selectors to provide a non null value"
-            )
+postOperation : GraphQL.Operation.Operation any GraphQL.Errors.Errors data -> (Result Http.Error (GraphQL.Response.Response GraphQL.Errors.Errors data) -> msg) -> Cmd msg
```

Comment:
Pourquoi avoir les deux postOperation si seulement une est exposée

Reply:
Bonne question, j'ai commencé par `postOperation` (pris essentiellement dans la doc `graphql-to-elm`) puis en voyant que les appels qu'on y faisait avaient beaucoup de code en commun j'ai créé `postOperationSimple` et remplacé les appels à `postOperation`. Cette dernière n'est donc plus appelée de l'extérieur du module, mais note qu'elle l'est par `postOperationSimple`. J'aurais pu inliner `postOperation` dans `postOperationSimple`, mais je me suis dit que c'était intéressant de distinguer l'appel lui-même de la couche de transformation des erreurs. Cela dit je ne suis pas attaché à cette séparation 🤷